php/templates/secret-busybox.yaml

```diff
@@ -11,7 +11,6 @@ metadata:
   {{- end }}
 type: Opaque
 data:
-  {{- range $k, $v := $secret }}
-  {{ $k }}: {{ tpl $v $root | b64enc | quote }}
-  {{- end }}
+  {{ $name }}: |
+{{ tpl $secret $root | indent 4 }}
```

Comment:
@cw-ozaki 
Without `base64env`,  users need to encode to base64 by themselfs?

Reply:
Oops. This is a mistake.